src/rabbit_control_main.erl

```diff
@@ -285,8 +285,8 @@ shutdown_node_and_wait_pid_to_stop(Node, Pid, Inform) ->
               "RabbitMQ node ~p running at PID ~s successfully shut down",
               [Node, Pid]),
             Res;
-        {error, Err} ->
```

Comment:
It depends on what exactly `rabbit:stop_and_halt/0` does. Our modification returned an error as a value. Both should be covered.